.pre-commit-config.yaml

```diff
@@ -31,22 +31,22 @@ repos:
       - id: check-merge-conflict
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.1.13
+    rev: v0.9.9
```

Comment:
```suggestion
    rev: v0.9.10
```
... like in SDK.

Reply:
was just released 3 days ago :D will update it